tests/unit/models/cart-model-test.js

```diff
@@ -82,6 +84,7 @@ test('it creates a line item when you add a variant', function (assert) {
   model.addVariants({ variant, quantity }).then(cart => {
     assert.equal(cart, model, 'it should be the same model, with updated attrs');
     assert.equal(cart.lineItems.length, 2);
+    assert.equal(cart.lineItemCount, 3);
```

Comment:
This should live in its own test. It shouldn't be interjected into all these other tests. We test length, because we know that's the output of adding a variant.

lineItemCount is a unique key, built on top cart. It should have a unique test instead of coupling to all of these.